res/values/strings.xml

```diff
@@ -465,6 +465,7 @@
     <string name="confirmation_remove_files_alert">Do you really want to remove the selected items?</string>
     <string name="confirmation_remove_folders_alert">Do you really want to remove the selected items and their contents?</string>
     <string name="maintenance_mode">Server in maintenance mode</string>
+    <string name="failed_upload_quota_exceeded_text">Not enough free space</string>
```

Comment:
agreed with the comments. it's important to clarify that the displayed problem is not caused by the device storage. "Account" or "Server" are both correct options IMHO.